src/test/java/wolox/training/utils/ValuesGenerator.java

```diff
@@ -24,7 +24,7 @@ private ValuesGenerator() {
      * @return A mocked {@link Book} id.
      */
     public static long validBookId() {
-        return Faker.instance().number().numberBetween(Long.MIN_VALUE, Long.MAX_VALUE);
+        return Faker.instance().number().numberBetween(1, Long.MAX_VALUE);
```

Comment:
What represents this '1'? 
Maybe moving to constant clarify this number represents.

Reply:
Okey, I'll extract it into a constant. Its just a minimum number for ids. It was changed to 1 because postgres'  `bigserial` type has values between 1 and `Long.MAX_VALUE` (https://www.postgresql.org/docs/10/datatype-numeric.html)